app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+    use Rack::Flash
+    get '/tweets' do
+        if logged_in?
+            @user = User.find_by_id(session[:user_id])
+            erb :'/tweets/tweets'
+        else
+            redirect to "/login"
+        end
+    end
+
+    get '/tweets/new' do
+        if logged_in?
+            erb :'/tweets/create_tweet'
+        else
+            redirect "/login"
+        end
+    end
+
+    post '/tweets' do
+        if !params[:content].empty?
+            @tweet = Tweet.create(:content => params[:content])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.